
export const databaseErrorInfo = () => {
    return 'Hubo un problema con la base de datos. Inténtalo de nuevo más tarde.';
  };
  
  export const authenticationErrorInfo = () => {
   return 'Hubo un problema con la autenticación. Inténtalo de nuevo más tarde.';
  };
  
  export const validationErrorInfo = () => {
   return 'Hubo un problema con la validación. Inténtalo de nuevo más tarde.';
  };
  